covid_xprize/validation/scenario_generator.ipynb

```diff
@@ -346,13 +359,105 @@
    "outputs": [],
    "source": [
     "from datetime import datetime\n",
-    "sd = 20200101  # IP file always contains data since inception\n",
+    "sd = start_date_str.replace('-', \"\")\n",
     "ed = end_date_str.replace('-', \"\")\n",
     "december_file_name = f\"../../../covid-xprize-robotasks/ips/live/{today}_{sd}_{ed}_ips.csv\"\n",
     "scenario_df.to_csv(december_file_name, index=False)\n",
     "print(f\"Saved to {december_file_name}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prescriptions"
```

Comment:
The 2 sections below are kind of unrelated: they generate scenarios for the prescription tasks.